docs/tutorial/voting-tutoral.md

```diff
@@ -0,0 +1,214 @@
+---
+title: How to deploy a voting smart contract Dapp
+sidebar_position: 1
+---
+
+In this video, we'll be deploying a voting smart contract Dapp on the Calimero Network.
+
+## Prerequisite
+
+To get started, ensure that you have the following:
+
+- Set up your [Calimero private shard](https://docs.calimero.network/getting_started/set_shard)
+- A code editor like [VSCode](https://code.visualstudio.com/download)
+- The [Calimero examples repository](https://github.com/calimero-is-near/calimero-examples) cloned onto your local machine
+- The [NEAR CLI](https://docs.near.org/tools/near-cli#setup) tool installed
+- Node and npm installed in your machine
+- The [Rust and WASM](https://docs.near.org/develop/contracts/introduction#rust-and-wasm) toolchain installed
+
+## Build smart contract
+
+- Clone the [Calimero examples repository](https://github.com/calimero-is-near/calimero-examples)
+
+```bash
+git clone https://github.com/calimero-is-near/calimero-examples
+```
+
+- cd to the voting directory.
+
+```bash
+cd calimero-examples/voting
+```
+
+- Navigate to `contracts/private`
+- Compile the Rust and WASM smart contract located using:
+
+```bash
+./build.sh
+```
+
+Compiled .wasm file of the contract located in :
+`target/wasm32-unknown-unknown/release/poll.wasm`
+
+![](../../static/img/voting_wasm.png)
+
+## Generating an auth token
+
+Before deploying the Voting smart dApp on our private shard, you must first generate a Calimero [auth token](/docs/getting_started/generate_token.md). This token will authenticate and authorize external applications to communicate with our shard.
+To create an auth token, follow these steps:
+
+- Log in to your Calimero Network account.
+- Click on the **Security** dropdown in the left navigation menu.
+- Select **Tokens**.
+- Click on **Create new token**.
+- Choose a name for your auth token.
+- Select a duration for the token. For this demo, we will use a Perpetual token.
+- Configure the token's permissions by selecting the appropriate checkboxes. In this case, we will select **Full access**.
```

Comment:
These are already described in the linked article, no need to repeat it here. Just mention that we issue a perpetual full access token